### PR TITLE
feat: remove npm Enterprise docs, in favor of npme.npmjs.com

### PR DESCRIPTION
**Pull request by bcoe**
This pull requests retires our documentation for npm Enterprise, instead linking to https://npme.npmjs.com which now has all of the Enterprise docs:

screen shot 2016-09-13 at 2 50 29 pm

_Note: I've added redirects from the old Enterprise docs, so that our old signup emails don't suddenly contain tons of 404s._

reviewers: @nexdrew, @ashleygwilliams

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
this all looks great. @nexdrew did you have any comments?

File: lib/redirects.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
 module.exports = {
-  '/enterprise/whitelist': '/enterprise/mirroring',
+  '/enterprise/whitelist': 'https://npme.npmjs.com/docs/workflow/mirroring.html',
+  '/enterprise/index': 'https://npme.npmjs.com',
+  '/enterprise/intro': 'https://npme.npmjs.com',
+  '/enterprise/requirements': 'https://npme.npmjs.com/docs/up-and-running/requirements.html',
+  '/enterprise/installation': 'https://npme.npmjs.com/docs/up-and-running/',
+  '/enterprise/client-configuration': 'https://npme.npmjs.com/docs/cli/',
+  '/enterprise/using-it': 'https://npme.npmjs.com',
```
Comment (nexdrew) [inline lib/redirects.js]: `/enterprise/using-it` could redirect to `https://npme.npmjs.com/docs/workflow/` instead

File: lib/redirects.js [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+  '/enterprise/mirroring': 'https://npme.npmjs.com/docs/workflow/mirroring.html',
+  '/enterprise/github': 'https://npme.npmjs.com/docs/up-and-running/auth/github.html',
+  '/enterprise/updating': 'https://npme.npmjs.com',
```
Comment (nexdrew) [inline lib/redirects.js]: `/enterprise/updating` could redirect to `https://npme.npmjs.com/docs/troubleshooting/faq.html#how-do-i-upgrade-npm-enterprise` instead

**Comment by ashleygwilliams** [comment]
@nexdrew's comments seem good- i'm happy with landing this after those!

**Pull request merged**